src/controller.cpp

```diff
@@ -1317,10 +1317,18 @@ void controller::edit_urls_file() {
 	v->push_empty_formaction();
 	stfl::reset();
 
-	utils::run_interactively(cmdline, "controller::edit_urls_file");
+	int editor_exit_code = utils::run_interactively(cmdline, "controller::edit_urls_file");
+	if (editor_exit_code != 0) {
+		LOG(level::DEBUG, "utils::run_interactively: "
```

Comment:
This line repeats a lot. Let's move it into `utils::run_interactively`! Don't forget to rephrase it (s/editor/command/ or something).